packages/common-all/src/store/NoteStore.ts

```diff
@@ -1,58 +1,48 @@
+import _ from "lodash";
+import { URI, Utils } from "vscode-uri";
+import { ERROR_SEVERITY, ERROR_STATUS } from "../constants";
+import { NoteUtils } from "../dnode";
+import { DendronError } from "../error";
 import {
-  DendronError,
   Disposable,
   DNoteLoc,
-  ERROR_SEVERITY,
-  ERROR_STATUS,
   FindNoteOpts,
-  genHash,
-  IDataStore,
-  IFileStore,
-  INoteStore,
-  isNotUndefined,
   NoteProps,
   NotePropsMeta,
-  NoteUtils,
   RespV3,
   WriteNoteMetaOpts,
   WriteNoteOpts,
-} from "@dendronhq/common-all";
-import { createDisposableLogger, DLogger } from "@dendronhq/common-server";
-import _ from "lodash";
+} from "../types";
+import { genHash, isNotUndefined } from "../utils";
+import { VaultUtils } from "../vault";
+import { IDataStore } from "./IDataStore";
+import { IFileStore } from "./IFileStore";
+import { INoteStore } from "./INoteStore";
 
 /**
  * Responsible for storing NoteProps non-metadata and NoteProps metadata
  */
 export class NoteStore implements Disposable, INoteStore<string> {
   private _fileStore: IFileStore;
   private _metadataStore: IDataStore<string, NotePropsMeta>;
-  private _wsRoot: string;
-  private _logger: DLogger;
-  private _loggerDispose: () => any;
-
-  constructor(opts: {
-    fileStore: IFileStore;
-    dataStore: IDataStore<string, NotePropsMeta>;
-    wsRoot: string;
-  }) {
-    this._fileStore = opts.fileStore;
-    this._metadataStore = opts.dataStore;
-    this._wsRoot = opts.wsRoot;
-    const { logger, dispose } = createDisposableLogger();
-    this._logger = logger;
-    this._loggerDispose = dispose;
+  private _wsRoot: URI;
+
+  constructor(
+    fileStore: IFileStore,
+    dataStore: IDataStore<string, NotePropsMeta>,
+    wsRoot: URI
+  ) {
+    this._fileStore = fileStore;
+    this._metadataStore = dataStore;
+    this._wsRoot = wsRoot;
   }
 
-  dispose() {
-    this._loggerDispose();
-  }
+  dispose() {}
```

Comment:
why was disposable removed?

Reply:
Current logger implementation in common-server requires fs, so the logger had to be removed from `NoteStore` for now to put it in common-all.  It can be added back later once we have an fs-agnostic logging interface.